framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -225,6 +225,13 @@ bool tlvTestVarList::add_complex_list(std::shared_ptr<cInner> ptr) {
     return true;
 }
 
+bool tlvTestVarList::isPostInitSucceeded() {
+    if( !is_var1_initialized() ){
```

Comment:
Please remove the spaces between the `if` brackets to the internal condition, and add space between the `if` to the open bracket.